cmd/k8s-operator/manifests/deploy/crds/tailscale.io_connectors.yaml

```diff
@@ -0,0 +1,89 @@
+apiVersion: apiextensions.k8s.io/v1
```

Comment:
Autogenerated from Go types in kube/apis/v1alpha1/types_connector.go.
There will be a make target for that and a test that it's up to date with Go types